packages/ses/README.md

```diff
@@ -272,6 +272,63 @@ The compartment will call `execute` with:
 method of third-party static module records to return promises, to support
 top-level await.
 
+### Transforms
+
+The `Compartment` constructor accepts a `transforms` option.
+This is an array of JavaScript source to source translation functions,
+in the order they should be applied.
+The input and output must both be valid JavaScript "Program" grammar
+constructions, code that is valid in a `<script>`, not a module.
```

Comment:
This is effective for the point you're making, but is not actually true. Only the output of the transform must be a Program. In the tildot example, text using `~.` is not a valid Program for the current standard definition of Program

Reply:
Ah, that’s true. A single transform (that can be a composite of other transforms) would be a great deal easier to explain than an array of transforms, since the array carries the implication that the input and output of each transform have the same “type”, which misleads to the conclusion that they are order-independent. They _are_ order-independent only insofar as the input and output grammar productions are the same.

Reply:
> Passing the source to the first function's input, then from each function's
> output to the next's input, the final function's output must be a valid
> JavaScript "Program" grammar construction, code that is valid in a `<script>`,
> not a module.